examples/regression/4-tutorials/plot_ccp_tutorial.py

```diff
@@ -51,6 +53,14 @@
 
 Recall that the ``alpha`` is ``1 - target coverage``.
 
+Warning:
+
+In this tutorial, we use ``unsafe_approximation=True`` to have a faster
+computation (because Read The Docs examples require fast computation).
+This mode use an approximation, which make the inference (``predict``) faster,
+but induce a small miscoverage. It is recommanded not to use it, or be
+very careful and empirically check the coverage and a test set.
+
```

Comment:
I needed to temporally use the old method (with ``unsafe_approximation=True``), which doesn't have the theoretical guarantees, otherwise the documentation was in timeout. I will need to change the dataset, to have faster computation, or transform this .py tuto in the doc, into a true notebook, which is not run during the documentation build.
Indeed, showing in the first tuto, the deprecated method which doesn't have guarantees is not a good idea.